packages/react/package.json

```diff
@@ -107,7 +106,6 @@
     "lodash.isobject": "^3.0.2",
     "react-compiler-runtime": "^19.1.0-rc.2",
     "react-intersection-observer": "^9.16.0",
-    "react-is": "^18.2.0",
```

Comment:
Why did you move this to devdependencies? It is imported and used in the components code